azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -37,10 +93,21 @@ func resourceArmKeyVaultCertificate() *schema.Resource {
 				ValidateFunc: azure.ValidateKeyVaultChildName,
 			},
 
+			"vault_id": {
```

Comment:
to better match the other resources I feel like this could make more sense as `key_vault_id`?

Reply:
Should we change `key_vault.vault_uri `key_vault.key_vault_uri` as well then?

Reply:
could do - but it's called `vault_uri` in the API (whereas the other is a Terraform specific field) - so I feel like it'd be best not too - WDYT?

Reply:
It seems inconsistent to have one be `key_vault_id` and one `vault_uri` when the are essential the same